### PR TITLE
Notifications returns period options - January

### PR DESCRIPTION
**Pull request by jonathangoulding**
https://eaflood.atlassian.net/browse/WATER-4716

As part of the ongoing work to migrate the legacy UI we are replacing the notification journey from the UI and rebuilding in system.

This change adds the notifications returns periods option for between january.

The other options will be added in following commits.

The submit logic will be developed in a separate commit so the 'values' used to capture the selected options may change.

### CODE REVIEW

File: app/presenters/notifications/setup/returns-period.presenter.js [hunk 1]
```diff
@@ -18,9 +18,44 @@ function go() {
...
+
   return []
 }
 
+/*
+ *  Checks if a date is in January
+ *  A date is in January if it is between 1st January - 28th January
+ */
+function _dayIsInJanuary(date) {
+  return date.getMonth() === 0 && date.getDate() >= 1 && date.getDate() <= 28
```
Comment (Cruikshanks) [inline app/presenters/notifications/setup/returns-period.presenter.js:37]: Really, _really_ minor NIT. But do we need to check `date.getDate() >= 1` _and_ `date.getDate() <= 28`. Wouldn't that last check suffice?

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
I just might not be seeing the possible scenario 🤷

File: app/presenters/notifications/setup/returns-period.presenter.js (hunk 1 above)
Comment (jonathangoulding) [reply app/presenters/notifications/setup/returns-period.presenter.js:37]: Good point. can remove.

**Review by Cruikshanks — DISMISSED** [review]

**Review by Cruikshanks — APPROVED** [review]
I'm not approving it again. Take it or leave it! 😁

**Pull request merged**